charts/backstage/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.17.0
```

Comment:
please run `pre-commit` or apply diff from the CI. New version should be propagated to `README.md` as well.